src/chrome/local-runtime.ts

```diff
@@ -208,6 +211,34 @@ export default class LocalRuntime {
     }
   }
 
+  // Returns the S3 url or local file path
+  async returnPDF(): Promise<string> {
+    const data = await pdf(this.client)
+
+    // check if S3 configured
+    if (process.env['CHROMELESS_S3_BUCKET_NAME'] && process.env['CHROMELESS_S3_BUCKET_URL']) {
+      const s3Path = `${cuid()}.pdf`
```

Comment:
I think this should be left alone in this pull request, and this can be addressed as part of #113.